core/src/integration-test/java/com/scalar/db/transaction/jdbc/JdbcTransactionAdminRepairTableIntegrationTest.java

```diff
@@ -27,4 +42,15 @@ protected AdminTestUtils getAdminTestUtils(String testName) {
   protected boolean hasCoordinatorTables() {
     return false;
   }
+
+  @Override
+  protected void waitForDifferentSessionDdl() {
+    // TODO: Revisit this naive workaround.
+    if (rdbEngine.toString().toLowerCase().contains("yugabyte")) {
```

Comment:
This class can't access `com.scalar.db.storage.jdbc.RdbEngineYugabyte`...

@brfrn169 Can we discuss this workaround later?

Reply:
Per the discussion, we decided the next plan:
1. Merge this backport for branch `3`
2. Add `isYugabyte()` method to `JavaTestUtil` or something and update the IT test code of `master` branch
3. Backport 2 to branch `3`